lib/chef/knife/bootstrap.rb

```diff
@@ -88,7 +88,7 @@ def self.friendly_opt_list(opt_array)
       option :winrm_auth_method,
         short: "-w AUTH-METHOD",
         long: "--winrm-auth-method AUTH-METHOD",
-        description: "The WinRM authentication method to use. Valid choices are #{friendly_opt_list(WINRM_AUTH_PROTOCOL_LIST)}.",
```

Comment:
I get why we're doing this here, but we should get a todo to fix the output of "in" cleaned up since it's not very user friendly.

Reply:
Also you might be able to kill that friendly_opt_list method now. I'm not sure if I was using that anywhere else